plugins/azure-devops/src/components/PullRequestsPage/PullRequestsPage.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Content,
+  Header,
+  Page,
+  Progress,
+  ResponseErrorPanel,
+} from '@backstage/core-components';
+import { PullRequestGroup, PullRequestGroupConfig } from './lib/types';
+import React, { useEffect, useState } from 'react';
+import { getCreatedByUserFilter, getPullRequestGroups } from './lib/utils';
+import { useDashboardPullRequests, useUserEmail } from '../../hooks';
+
+import { DashboardPullRequest } from '@backstage/plugin-azure-devops-common';
+import { PullRequestGrid } from './lib/PullRequestGrid';
+
+function usePullRequestGroupConfigs(
```

Comment:
This hook feels a bit odd tbh, shouldn't it just be a normal function that transforms a user email into the config array directly?

Reply:
I will look into this. I'm quite new to React so still learning the ropes at the moment. I will fix this in a follow up PR as it was a change from my previous PR. I might ask you a few more questions around this on discord.